### PR TITLE
feat(cloudshell script): improve message in case of private endpoint

### PR DESCRIPTION
**Pull request by mercoglianoisidoro**
This PR improve the message in case of eks private endpoint

Here an example of the provided output:

image

### CODE REVIEW

File: cloudshell/execute_on_cloudshell.sh [hunk 1]
```diff
@@ -81,8 +89,12 @@ prepare_execution(){
       echo -e "Connection establised (kubectl get nodes command executed successfully)"
       set -e
   else
-      echo -e "Error: impossible to communicate with the cluster (kubectl get nodes command failed)\n
+      echo -e "\nError: impossible to communicate with the cluster (kubectl get nodes command failed)
       Please verify connectivity or access configuration.\n" >&2
+      if [ "$endpointPublicAccess" = "false" ]; then
+      echo -e "\n As public endpoints are closed for this cluster, please report this message to DoiT: an alternative approach will be provided.\n
```
Comment (migueldelucasdoit) [inline cloudshell/execute_on_cloudshell.sh]: Which alternative approach? IMHO is better just to say `As public endpoints are closed for this cluster, please report this message to DoiT.`

**Review by migueldelucasdoit — APPROVED** [review]
LGTM. Just added a comment.

File: cloudshell/execute_on_cloudshell.sh (hunk 1 above)
Comment (mercoglianoisidoro) [reply cloudshell/execute_on_cloudshell.sh]: Which alternative approach?=> 
I'm working on finding the [best solution for private endpoints](https://doitintl.atlassian.net/wiki/spaces/CRE/pages/851378201/EKS+Private+Endpoints).
The CRE could propose an alternative approach.

Anyway, ok for your suggestion.

**Comment by mercoglianoisidoro** [comment]
after feedbacks, current message is the following:
image

**Pull request merged**